lib/driver.js

```diff
@@ -227,6 +237,9 @@ class AndroidDriver extends BaseDriver {
       }
     });
 
+    // Let's try to unlock the device
+    await helpers.unlock(this, this.adb, this.caps);
```

Comment:
Imagine that `this.useUnlockHelperApp` is true. Now we've gotten to this line. I'm a little confused why we call `helpers.unlock` here, which calls `helpers.unlockWithHelperApp` inside of it, even though we've already called it above in this case?